GameFramework/RotaionValue/Degree.h

```diff
@@ -0,0 +1,185 @@
+﻿//----------------------------------------------------------
+// <filename>Degree.h</filename>
+//----------------------------------------------------------
+
+#ifndef DEGREE_H
+#define DEGREE_H
+
+#include <Window.h>
```

Comment:
自動テスト用プロジェクトを作っていて気付きましたが、これって自作のヘッダですよね？
であれば
```suggestion
#include "Window.h"
```
ですね。

Reply:
Windows.hのつもりが間違っていましたしかし現在不要なため削除しました